api-specificatie/openapi.yaml

```diff
@@ -1346,19 +1340,6 @@ components:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
           description: 'De aan het openbareruimte gerelateerde woonplaats (link).'
 
-    Nummeraanduiding_embedded:
-      type: object
-      properties:
-        adres:
-          $ref: '#/components/schemas/AdresHalBasis'
-          description: 'De aan de nummeraanduiding gerelateerde adres (object).'
-        openbareRuimte:
```

Comment:
Ik zie in #113 niet staan dat de gerelateerde objecten openbare ruimte en woonplaats niet ge-expand kunnen worden. Er staat bij nummeraanduidingen alleen iets over de fields parameter. Het gebruik van de expand parameter op _embedded parameters is toch gebruikelijk? Waarom moeten hier de gerelateerde objecten uit _embedded worden verwijderd?

Reply:
Ha Mark, het is de bedoeling dat woonplaats, nummeraanduiding en openbare ruimte geexpand kunnen worden bij adres, zodat gebruikers die behoefte hebben aan BAG details (brondocumenten, besluitinfo etc. denk aan de advocaten, bezwaarcommissies etc.) deze tegelijk met het adres kunnen opvragen. 
Er is mij geen behoefte bekend om de gerelateerde objecten van deze drie resources (nummeraanduiding in het bijzonder) zelf ook te expanden.